src/share/infrastructure/prisma.ts

```diff
@@ -0,0 +1,19 @@
+import { PrismaClient } from '@prisma/client';
+
+let prismaInstance: PrismaClient | null = null;
+
+export const getPrismaClient = (): PrismaClient => {
```

Comment:
getPrismaClient returns a PrismaClient singleton but there's no registration for process shutdown to gracefully disconnect. Also no explicit pool/tls/config options are set (may be acceptable but worth making explicit).

**Sugerencia:**
```typescript
Consider adding a small lifecycle handling block (or document usage) to disconnect Prisma on SIGTERM/SIGINT during server shutdown. Example:
process.on('SIGTERM', async () => { await closePrismaClient(); process.exit(0); });
Also consider passing connection options if needed (log levels, error handling). Priority: minor
```